### PR TITLE
Task navbar

### PR DESCRIPTION
**Pull request by TimTapley**
Created nav bar html and added css styling.

### CODE REVIEW

**Comment by mclack** [comment]
Unnecessary flex-direction (row is default)
Justify-content conflict (space-between, center)
More styling for the <a> tags to make it look more like a finished product

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+}
+/*
+img {
+	max-height: 500px;
+	max-width: 500px;
+}
+*/
+
+nav {
+	background-color: #95CDED;
```
Comment (HannaSBarratt) [inline styles.css]: this isn't necessary as you have set the background-color and color on the parent element body

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+}
+@media screen and (max-width: 900px) {
+   
+	@media screen and (max-width: 900px) {
+	  .wide-nav {
+	    display: none;
+	  }
+
+	  .narrow-nav {
```
Comment (HannaSBarratt) [inline styles.css:88]: this button probably needs more styling, and the position on the page doesn't feel quite right yet (unless this is in line with your wireframe - I don't have it)

**Review by HannaSBarratt — COMMENTED** [review]
hey tim, looks good! i've put a few comments as comments in the code (in files changed).

File: styles.css (hunk 2 above)
Comment (HannaSBarratt) [reply styles.css:88]: looks good and in a good position now! you might want to add some padding or margin so it is in line with your wide-nav element.

**Pull request merged**